server/src/utils/const.ts

```diff
@@ -0,0 +1,27 @@
+/*!
+ * Copyright 2022 - Swiss Data Science Center (SDSC)
+ * A partnership between École Polytechnique Fédérale de Lausanne (EPFL) and
+ * Eidgenössische Technische Hochschule Zürich (ETHZ).
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+// Prefix of the last project data
+const LPROJECTS_PREFIX = "LPROJECT_";
+const LPROJECTS_DEFAULT_LENGTH = 100;
```

Comment:
I think we can make this default smaller, say 20.
```suggestion
const LPROJECTS_DEFAULT_LENGTH = 20;
```